pkg/resource/deploy/step.go

```diff
@@ -1206,11 +1209,12 @@ func NewImportStep(deployment *Deployment, reg RegisterResourceEvent, new *resou
 		new:           new,
 		ignoreChanges: ignoreChanges,
 		randomSeed:    randomSeed,
+		cts:           cts,
```

Comment:
Is `cts` allowed to be `nil`? If so, when? If e.g. it's only for `pulumi import`-style imports, maybe we document that/fold that into the `planned` thing above.